hack/prometheus-operator-crd-cluster-roles.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prometheus-crd-view
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+- apiGroups: ["monitoring.coreos.com"]
```

Comment:
this doesn't match the roles in the manifest file. Do we need this since we should be using the manifest files?